src/datasets/iterable_dataset.py

```diff
@@ -1118,22 +1148,25 @@ def __init__(
         ex_iterable: _BaseExamplesIterable,
         info: Optional[DatasetInfo] = None,
         split: Optional[NamedSplit] = None,
-        format_type: Optional[str] = None,
+        formatting: Optional[FormattingConfig] = None,
         shuffling: Optional[ShufflingConfig] = None,
         distributed: Optional[DistributedConfig] = None,
         token_per_repo_id: Optional[Dict[str, Union[str, bool, None]]] = None,
+        format_type="deprecated",
```

Comment:
Nice you deprecate it. What about in the classes above?
- MappedExamplesIterable
- FilteredExamplesIterable